CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ Deprecated `neofs_node_object_epoch` metric for IR and SN (the same for both)
 has been removed. Use `neofs_node_state_epoch` for SN and `neofs_ir_state_epoch`
 for IR instead.
 Deprecated `--no-precheck` flag in `neofs-cli container set-eacl` use `--force` flag instead for skipping validation checks.
+Unified usage of `--expired-at` and `--lifetime` flags for `neofs-cli`.
```

Comment:
usually this section contans info that "speaks" with a reader, like "we have done (in non-compatible way) **that** so update **that** cause it can be broken now"

so i would either describe which cmds have a new flags now, or move it to `Changed` or `Added` section